MIGRATION.md

```diff
@@ -1,10 +1,16 @@
-## Migration Guide
+# Migration Guide
 
-### Migrating from v5 to v6
+## Migrating from v5 to v6
 
 v6 of o-buttons simplifies the `o-buttons` interface, encourages CSS reuse, and makes custom buttons consistent with default buttons.
 
-#### Markup
+### Updated dependencies
+
+The dependencies for this component have all been updated to the latest major versions.
+If you have any conflicts while installing this version, you'll need to first update
+its dependencies. See [the Bower config for these](./bower.json).
```

Comment:
we should make this link be a link to a specific commit, if we don't do that then when v7 of o-buttons is released, this migration guide for v5->v6 will point to the wrong bower.json

Reply:
We won't know the commit hash until the `breaking` branch has been merged into master, as we're using a rebase. So I think once we've got all of the versions out we should go back and correct the link in all these migration files. Not sure where the best place to create an issue for this is – suggestions?

For now going to merge this so I can get the major released.